website/static/js/contribRemover.js

```diff
@@ -0,0 +1,294 @@
+/**
+ * Controller for the Add Contributor modal.
+ */
+'use strict';
+
+var $ = require('jquery');
+var ko = require('knockout');
+var bootbox = require('bootbox');
+var Raven = require('raven-js');
+
+var oop = require('./oop');
+var $osf = require('./osfHelpers');
+var Paginator = require('./paginator');
+
+var API_BASE = 'http://localhost:8000/v2/nodes/';
```

Comment:
> I don't find instances of this pattern elsewhere in the codebase.

But I *do* find an [instance](https://github.com/CenterForOpenScience/osf.io/commit/7c417f119c78f96146f906a7225f64b168d8c855#diff-4f5e70a2bdc38e3df54149befd145224L20) of this pattern that was weeded out in a previous code review. ;-)